src/Controls/tests/TestCases.HostApp/Issues/Issue26810.xaml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Maui.Controls.Sample.Issues.Issue26810">
+    <Grid RowDefinitions="100,100,*"
+          AutomationId="26810MainGrid">
+        <Button Text="Scroll to Random Item"
+                AutomationId="26810Button"
+                x:Name="ScrollToButton"
+                Grid.Row="0"
+                Clicked="ScrollToButton_Clicked"
+                HorizontalOptions="Center"
+                VerticalOptions="Center"/>
+        <Button Text="Add New Item"
```

Comment:
Could you include a Button to modify the ItemsUpdatingScrollMode property and test add items with the different options?

Reply:
Yes @jsuarezruiz , We have included all options for the `ItemsUpdatingScrollMode` property and have also added test cases to cover the horizontal orientation.